webapp/src/assets/svg/MoonIcon.vue

```diff
@@ -19,7 +20,7 @@ export default {
   name: 'MoonIcon',
   props: {
     customClass: {
-      default: 'w-6 h-6',
```

Comment:
Why add the bootstrap icon class? It doesn't seem to do anything when I remove them